### ISSUE TITLE
KeyError: 'Objective (impartial and unbiased presentation of facts and findings)'

### ISSUE THREAD

**Issue by 01110011011101010110010001101111** [assafelovic/gpt-researcher#668]
Maybe I’m missing something but when running [this Colab](https://colab.research.google.com/github/assafelovic/gpt-researcher/blob/master/examples/pip-run.ipynb) without any changes besides updating the keys, I’m getting the error:

```
KeyError: 'Objective (impartial and unbiased presentation of facts and findings)'
```

It is thrown from this line:

```
report = asyncio.run(get_report(query, report_type))
```

Thanks in advance!

**Comment by 0x11c11e** [comment]
Thank you for reporting this. It seems that GPTResearcher requires a Tone argument but should default to the Objective tone if none is provided. There appears to be a bug preventing it from picking up the default value. I will create a branch to address this issue immediately. In the meantime, you can proceed by explicitly adding a Tone argument:

`from gpt_researcher.utils.enum import Tone`
`GPTResearcher(query, report_type, tone=Tone.Objective)`

**Comment by assafelovic** [comment]
Thank you for this @0x11c11e !

**Comment by ElishaKay** [comment]
Fixed in #674
